apps/component-date/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const summary = require('hof-behaviour-summary-page');
+
+module.exports = {
+  name: 'component-date',
+  baseUrl: '/component-date',
+  steps: {
+    '/date-of-birth': {
+      fields: ['date-of-birth'],
+      next: '/passport-expiry'
+    },
+    '/passport-expiry': {
+      fields: ['passport-expiry'],
+      next: '/confirm'
```

Comment:
When the steps are ordered in the ordered sequentially like this do you need to declare the `next` step or is it implicit?